pyshorteners/shorteners/bitly.py

```diff
@@ -63,16 +71,21 @@ def expand(self, url):
         Raises:
             ExpandingErrorException: If the API Returns an error as response
         """
-        expand_url = f'{self.api_url}v3/expand'
-        params = {
-            'shortUrl': url,
-            'access_token': self.api_key,
-            'format': 'txt',
-        }
-        response = self._get(expand_url, params=params)
-        if response.ok:
-            return response.text.strip()
-        raise ExpandingErrorException(response.content)
+        self.clean_url(url)
+        url = "".join(urlparse(url)[1:3])
```

Comment:
this part doesn't sound right. Are you trying to get just the path for the url id?

Reply:
yeah I used the approach mentioned [here](https://github.com/ellisonleao/pyshorteners/pull/131#discussion_r332306419)
sadly the API doesn't accept the full link anymore

Reply:
can't you just use the bit.ly id (e.g "http://bit.ly/1234" => "1234") in the url param?

Reply:
no
the domain is something pro users can change so what bit.ly/1234 will expand to will not be what exampledomain.com/1234 will expand to